### DISCUSSION TITLE
Replace Tilde(~) in NFPMS version with exact git tag

### DISCUSSION THREAD

**Discussion by rupamkhaitan** [goreleaser/goreleaser#1630] [Q&A] [answered]
@caarlos0 is there any way i can fix the version string shown in dpkg with my exact tag??

Say my tag is 1.0.0-b1-noc and i made a release build of debian and install in the linux (say my debian pkg name is go-broker)

When i install it

```
dpkg -i go-broker
```

After installing the debian , I match the version and see that its replaced hyphen(-) with tilde(~)
```
dpkg -s go-broker
```

![image](https://user-images.githubusercontent.com/3277629/86041738-ba3a2200-ba0b-11ea-8ee8-63d1e26dff4c.png)

**Comment by caarlos0** [comment, marked as answer]
the tilde there is how debian versioning works, not possible to change.

check https://www.debian.org/doc/manuals/maint-guide/first.en.html#namever and also https://github.com/goreleaser/nfpm/pull/111
